src/trans-word-level/sequence.cpp

```diff
@@ -132,7 +132,31 @@ std::vector<std::pair<mp_integer, exprt>> instantiate_sequence(
   else if(expr.id() == ID_sva_sequence_intersect)
   {
     // IEEE 1800-2017 16.9.6
-    PRECONDITION(false);
+    // - Both the operands shall match.
+    // — The lengths of the two matches of the operand sequences shall be the same.
```

Comment:
Where does the code enforce these two preconditions? It seems the code below is missing an else branch?!

Reply:
Clarified the comment; this is not a precondition, but the condition for the intersection to match.